conductor/blueprints/package/controllers.py

```diff
@@ -103,6 +105,31 @@ def toggle_publish(name, token, toggle=False, publish=False):
     return {'success': True, 'published': not private}
 
 
+def update_params(name, token, params):
+    '''
+    Update package.defaultParams for the passed `name`. Only the owner can
+    update.
+    '''
+    try:
+        token = jwt.decode(token.encode('ascii'),
+                           PUBLIC_KEY,
+                           algorithm='RS256')
+        userid = token['userid']
+        if name.split(':')[0] != userid:
+            logging.error('USERID=%r, name=%r', userid, name)
+            return None
+    except jwt.InvalidTokenError:
+        return None
+
+    _, _, datapackage, _, _, _, _, _ = package_registry.get_raw(name)
```

Comment:
I think you're able to do like `_, _, datapackage, *_ = ...`, so it's a bit less entangled.